client/hwlib/src/py_bindings.rs

```diff
@@ -18,24 +18,32 @@
  *        Nadzeya Hutsko <nadzeya.hutsko@canonical.com>
  */
 
-use crate::get_certification_status as native_get_certification_status;
+use crate::builders::request_builders::{create_certification_status_request, Paths};
+use crate::send_certification_status_request as native_send_certification_status_request;
 use once_cell::sync::Lazy;
 use pyo3::exceptions::PyRuntimeError;
 use pyo3::prelude::*;
 use pyo3::types::PyString;
 use pyo3::wrap_pyfunction;
 use pyo3::{PyObject, PyResult, Python};
+use serde_json;
 use tokio::runtime::Runtime;
 
-static RT: Lazy<Runtime> = Lazy::new(|| Runtime::new().expect("Failed to create Tokio runtime"));
+static RT: Lazy<Runtime> = Lazy::new(|| Runtime::new().expect("failed to create Tokio runtime"));
```

Comment:
This may not be relevant for this PR, but I think everywhere we see `static FOO: Lazy<T>`, this can be replaced with `lazy_static` as a convention